bsdlib/CHANGELOG.rst

```diff
@@ -5,6 +5,22 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.3.1
+************
+
+Updated library with various changes:
+
+* Corrected GNSS API to not fault if not read fast enough.
+* Improved length reporting on GNSS NMEA strings to report length until zero-termination.
+* Improved closing of GNSS socket. If closed, it will now also stop the GNSS from running.
+* Corrected bitmask value of NRF_GNSS_SV_FLAG_UNHEALTHY.
+* Added side API for APN Class management.
+* Removed NRF_SO_PDN_CLASS from nrf_socket.h as it is replaced by side API for APN class management.
+* Improved nrf_poll() error return on non-timeout errors to be NRF_EAGAIN, to align with standard return codes from poll().
+* Added implementation of inet_pton() and inet_ntop().
```

Comment:
@b-gent, sorry the two last bullets here was just added, it slipped in the first round. Could you take a second look?